src/Examples/DAE/petsc_pid_example.ipynb

```diff
@@ -0,0 +1,754 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.fs.tank.control_volume.properties_out[24].pressure)])\n",
+    "a = plt.ylabel(\"tank pressure (Pa)\")\n",
+    "a = plt.xlabel(\"time (s)\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e2fc8180-9ad7-4cf5-9aad-78b882730347",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
Is this a blank cell with no code? Is this intentional, for the user to write their own code/experiment a little? Sincere question -- I'm new to Jupyter.

Reply:
Maybe.  I think a blank cell on the end is normal.

Reply:
If you click "View file" it will open a GitHub rendered version of the notebook that is much easier to review.